.gitignore

```diff
@@ -1,10 +1,9 @@
 # Ignore artifacts generated by "pub".
+**/build/
 .packages
-**/build
-.dart_tool/
 pubspec.lock
 
 # Files generated by dart tools.
-.dart_tool
-/doc/api
-**/doc/api
+**/.dart_tool/
+**/doc/api/
+*.g.dart
```

Comment:
Maybe we should do `*_po.go.dart` instead? I could imagine using generated parts in other areas of this repo in the future...

Reply:
We already do for offline_urls.g.dart.

This doesn't prevent you from tracking or committing a `.g.dart` file, it just want show as unstaged _until_ you add it to be tracked.